tox.ini

```diff
@@ -16,6 +16,7 @@ setenv =
     PYTHONPATH = {toxinidir}
 commands =
     pip install -U pip
+    pip install -U git+https://github.com/nucleogenesis/le-utils.git@slideshow-content-type#egg=le-utils
```

Comment:
@nucleogenesis  You don't need this anymore --- so can remove when you next work on this PR